src/GraphQL.Conventions/Adapters/ResolutionContext.cs

```diff
@@ -61,7 +61,7 @@ public void SetArgument(string name, object value)
 
         public IUserContext UserContext => FieldContext.GetUserContext();
 
-        public IDependencyInjector DependencyInjector => FieldContext.GetDependencyInjector();
+        public IServiceProvider DependencyInjector => FieldContext.RequestServices;
```

Comment:
Rename?

Reply:
Ah, yep 👍 Go for it

Reply:
Renamed to ServiceProvider.  Alternate: `RequestServices`

Reply:
Thanks. `ServiceProvider` sounds good to me.